pkg/cli/predict.go

```diff
@@ -195,17 +207,39 @@ func predictIndividualInputs(predictor predict.Predictor, inputFlags []string, o
 		return err
 	}
 
-	if _, err := outFile.Write(out); err != nil {
+	if _, err := outFile.Write(output); err != nil {
 		return err
 	}
 	if err := outFile.Close(); err != nil {
 		return err
 	}
-
 	console.Infof("Written output to %s", outputPath)
 	return nil
 }
 
+func handleMultipleFileOutput(prediction *predict.Response, outputSchema *openapi3.Schema) error {
+	outputs, ok := (*prediction.Output).([]interface{})
+	if !ok {
+		return fmt.Errorf("Failed to decode output")
+	}
+
+	for i, output := range outputs {
+		outputString := output.(string)
+		dataurlObj, err := dataurl.DecodeString(outputString)
+		if err != nil {
+			return fmt.Errorf("Failed to decode dataurl: %w", err)
+		}
+		out := dataurlObj.Data
+		extension := mime.ExtensionByType(dataurlObj.ContentType())
+		outputPath := fmt.Sprintf("output.%d%s", i, extension)
```

Comment:
Do we wanna put some leading zeros on this file format so the files will sort alphabetically beyond 9? `output.000.png`